jmolecules-events/src/main/java/org/jmolecules/event/annotation/DomainEvent.java

```diff
@@ -29,4 +29,11 @@
  *      Reference (Evans) - Domain Events</a>
  */
```

Comment:
It looks like we're completely missing retention and target annotations here. 😬

Reply:
Added it. 

Since this is already in 1.0.0, we need a tool to check for such things so that we can't miss it ;)